apps/web-nextjs/README.md

```
@@ -1,17 +1,13 @@
+[![Web - E2E Tests](https://github.com/Stbs0/sudan-codex/actions/workflows/web-e2e.yml/badge.svg?branch=main)](https://github.com/Stbs0/sudan-codex/actions/workflows/web-e2e.yml) [![Web - Unit & Integration Tests](https://github.com/Stbs0/sudan-codex/actions/workflows/web-unit.yml/badge.svg)](https://github.com/Stbs0/sudan-codex/actions/workflows/web-unit.yml) ![Vercel Deploy](https://deploy-badge.vercel.app/vercel/sudan-codex-web)
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)